\subsection{DNA Cloning}
\subsubsection{PCR Amplification of Insert DNA}
Using the synthetic genes S-Tag-A and S-Tag-B (Subsection \ref{subseq:synthetic_genes}) and the primer pairs d29-A fwd / d29-A rev (with template S-Tag-A), S-Tag-A fwd / d29-A rev (with template S-Tag-A), and S-Tag-B fwd / S-Tag-B rev (with template S-Tag-B), polymerase chain reactions (PCRs) were conducted for amplification of the inserts. The primers also introduced overlapping regions with the plasmid pLenEx-Strep-eYFP for the following Gibson Assembly, and in the case of d29-A fwd, a start codon. The PCR was conducted with a Pfu polymerase. The composition of the PCR mix is listed in Table \ref{tab:methods_pcr_composition}.
\begin{table}
    \centering
    \caption{\textbf{Composition of the PCR Mix for Insert Amplification. }}
    \label{tab:methods_pcr_composition}
\begin{tabular}{ll}
\toprule
\textbf{Component} & \textbf{Amount} \\
\midrule
MQ-H\textsubscript{2}O & \SI{38}{\micro\litre} \\
10x Pfu Buffer & \SI{5}{\micro\litre} \\
dNTPs ($\SI{10}{\milli\Molar}$) & \SI{2}{\micro\litre} \\
Forward Primer (\SI{10}{\micro\Molar}) & \SI{2}{\micro\litre} \\
Reverse Primer (\SI{10}{\micro\Molar}) & \SI{2}{\micro\litre} \\
Pfu DNA Polymerase (TODO: Lookup activity) & \SI{0.5}{\micro\litre} \\
DNA template (TODO: Lookup concentration) & \SI{5}{\micro\litre}  \\
\bottomrule
\end{tabular}
\end{table}
The PCR reaction was conducted in a thermocycler. The exact program is described in Table \ref{tab:methods_pcr_cycle}. After running the PCR, the products were frozen at $\SI{-20}{\degreeCelsius}$ until further processing.
\begin{table}[ht]
    \centering
    \caption{\textbf{PCR Program used for Insert Amplification.}}
    \label{tab:methods_pcr_cycle}
    \begin{tabular}{lll}
    \toprule
    \textbf{Step} & \textbf{Temperature (Â°C)} & \textbf{Time} \\
    \midrule
    Initial Denaturation     & 94  & 4 min   \\[1ex]
    \multicolumn{3}{l}{Repeat for 30 cycles:} \\[1ex]
    \quad Denaturation       & 94  & 30 s    \\
    \quad Annealing          & 57  & 30 s    \\
    \quad Elongation         & 72  & 100 s   \\[1ex]
    Final Elongation         & 72  & 5 min   \\
    Storage                  & 8   & $\infty$ (hold)  \\
    \bottomrule
    \end{tabular}
\end{table}
\subsubsection{Plasmid Restriction Digest}
The plasmid pLenEx-Strep-eYFP was digested using the restriction enzymes NcoI-HF and KpnI-HF. Therefore, $\SI{10}{\micro\gram}$ of the plasmid, $\SI{5}{\micro\litre}$ CutSmart\textsuperscript{\textregistered} Buffer, $\SI{1}{\micro\litre}$ of NcoI-HF (TODO: Lookup activity), and $\SI{1}{\micro\litre}$ of KpnI-HF (TODO: Lookup activity) were added to MQ-H\textsubscript{2}O up to a total volume of $\SI{50}{\micro\litre}$. The mixture was incubated at $\SI{37}{\degreeCelsius}$ for three hours and thereafter frozen at $\SI{-20}{\degreeCelsius}$ until further processing. 

\subsubsection{Agarose Gel Electrophoresis and DNA Recovery}
Agarose gel electrophoresis was used to validate and purify the restricted plasmid and the PCR products. $\SI{50}{\micro\litre}$ of each sample were mixed with $\SI{5}{\micro\litre}$ 10x Sabu and applied to the gel, splitting the sample into $\SI{25}{\micro\litre}$ per track. The gels were run at either $\SI{100}{\volt}$ for about 45 minutes or at $\SI{120}{\volt}$ for about 60 minutes, depending on the size of the gel. After the gel ran through, the samples were cut out under illumination from a UV light source. The DNA was recovered from the gel samples using the Wizard\textsuperscript{\textregistered} SV Gel and PCR Clean-Up System. The steps were carried out following the manufacturer's protocol. DNA concentrations of each sample were determined using the NanoDrop\textsuperscript{\texttrademark} One.

\subsubsection{Gibson Assembly}
A Gibson Assembly using $\SI{50}{\nano\gram}$ of purified linear vector DNA was conducted to create the plasmids pLenEx-d29-A, pLenEx-S-Tag-A, and pLenEx-S-Tag-B. The required mass of insert DNA was calculated using Equation \ref{eq:gibson}.
\begin{equation}\label{eq:gibson}
\text{Mass}_{\text{insert}} = \left(\frac{\text{desired molar ratio}}{1}\right) \times \text{Mass}_{\text{vector}} \times \left(\frac{\text{Length}_{\text{insert}}}{\text{Length}_{\text{vector}}}\right)
\end{equation}
The molar ratio was chosen as 2. 
Given the plasmid length of 2173 bp, and the insert lengths of 679 bp, 802 bp, and 802 bp for the three inserts d29-A, S-Tag-A, and S-Tag-B respectively, the insert DNA masses were calculated as $\SI{31.2}{\nano\gram}$ for d29-A and $\SI{37}{\nano\gram}$ for S-Tag-A and S-Tag-B. 
The plasmid and inserts and $\SI{10}{\micro\litre}$ Gibson Assembly\textsuperscript{\textregistered} Master Mix were added to MQ-H\textsubscript{2}O to a total volume of $\SI{20}{\micro\litre}$ and incubated at $\SI{50}{\degreeCelsius}$ for one hour. 

\subsubsection{Transformation into Competent Cells}
Transformation of the assembled plasmids into NEB\textsuperscript{\textregistered} Turbo Competent \emph{E. coli} cells was carried out using the manufacturer's High Efficiency Transformation Protocol, using a reduced volume of cells compared to the original protocol.

After thawing the cells on ice for 10 minutes, the cells were gently mixed and \SI{20}{\micro\liter} of the cells were transferred to a reaction tube on ice. TODO \SI{0}{\nano\gram} of plasmid DNA were added to the cell mixture and carefully flicked to mix cells and DNA. The mixture was placed on ice for 30 minutes. Afterwards, a heat shock at exactly \SI{42}{\celsius} was conducted for 30 seconds. The cells were thereafter placed on ice for 5 minutes. Afterwards, \SI{950}{\micro\liter} of room temperature salt medium was added to the mixture. The cells were rotated at \SI{37}{\celsius} for 60 minutes, during which LB-Amp selection plates were warmed to \SI{37}{\celsius}. The cells were mixed thoroughly by flicking the tube and inverting it. Afterwards, \SI{100}{\micro\liter} were applied to a selection plate and incubated overnight at \SI{37}{\celsius}.

\subsubsection{Colony PCR}
After incubation overnight, 9 colonies of each construct were picked for Colony PCR. The master mix for the PCR was created following the composition in \autoref{tab:pcr-mix}.

\begin{table}[h]
\centering
\caption{PCR Master Mix Composition}
\label{tab:pcr-mix}
\begin{tabular}{@{}ll@{}}
\toprule
Component & Amount \\ 
\midrule
MQ-H\textsubscript{2}O & \SI{15}{\micro\liter} \\
5x Green GoTaq\textsuperscript{\textregistered} Reaction Buffer & \SI{4}{\micro\liter} \\
dNTPs (\SI{10}{\milli\Molar}) & \SI{0.4}{\micro\liter} \\
Forward Primer & \SI{0.2}{\micro\liter} \\
Reverse Primer & \SI{0.2}{\micro\liter} \\
Taq DNA Polymerase & \SI{0.2}{\micro\liter} \\
\bottomrule
\end{tabular}
\end{table}

For each sample, \SI{20}{\micro\liter} of the master mix were transferred into PCR tubes. The colonies were picked using sterile toothpicks and applied to an LB-Amp reference plate. Afterward, they were placed into the PCR tubes for about 30 seconds. The reference plate was incubated at \SI{37}{\celsius} overnight. The PCR was run using the program from \autoref{tab:methods_pcr_cycle}.
\subsubsection{Plasmid Mini-Preparation and Sequencing}
\subsubsection{Plasmid Midi-Preparation}

\subsection{Protein Expression and Purification}
\subsubsection{Cell-Free Protein Expression}
\subsubsection{Protein Purification Using Capto Core 700}

\subsection{Protein Analysis}
\subsubsection{SDS-PAGE}
\subsubsection{Coomassie Staining}
\subsubsection{Western Blot}
\subsubsection{ELISA}
\subsubsection{Electron Microscopy}